js/danoni_main.js

```
@@ -4838,76 +4867,123 @@ const titleInit = () => {
+				if (val === g_settings.musicIdxNum) {
+					g_headerObj.viewLists.push(j);
+					tmpCreatorList.push(g_headerObj.creatorNames[j]);
+				}
+			});
+			divRoot.appendChild(drawBackArrow(g_headerObj.viewLists[0] + 1));
+			loadLocalStorage(g_settings.musicIdxNum);
+			makeInfoWindow(g_msgInfoObj.W_0041.split(`{0}`).join(g_localStorageUrl), ``,
+				{ _backColor: `#333333`, _textColor: `#cccccc`, _pointerEvents: C_DIS_INHERIT });
+		}
+	} else if (!g_headerObj.customTitleArrowUse) {
+		divRoot.appendChild(drawBackArrow());
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)